src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -249,6 +251,39 @@ def load_command_table(self, _):
         g.generic_update_command('update',
                                  custom_func_name='db_audit_policy_update')
 
+    database_long_term_retention_policies_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#BackupLongTermRetentionPoliciesOperations.{}',
+        client_factory=get_sql_database_long_term_retention_policies_operations)
+
+    with self.command_group('sql db ltr-policy',
```

Comment:
Can we add is_preview=True for the two command group?

Reply:
done